src/layers/legacy/medRegistration/itkProcessRegistration.cpp

```diff
@@ -203,7 +203,7 @@ template <typename PixelType>
             }
             catch(itk::ExceptionObject &ex)
             {
-                dtkDebug() << "Extraction failed";
+                dtkDebug() << "Extraction failed" << ex.what();
```

Comment:
small comment, but there is a ":" missing here

Reply:
It's done.